.git.docker

```diff
@@ -1 +1 @@
-eea39f7cb8dd3f1267764924b0d9a1ddae6c1428
+5ef8a2280b17d8c76d79bf427ab63f7ad7af55bd
```

Comment:
please, use tag, not hash
hash is not recommended even by our public documentation